src/grid/grid-cell.js

```diff
@@ -34,11 +37,12 @@ class GridCell extends React.Component {
 
   render() {
     return (
-      <div
+      <div // eslint-disable-line jsx-a11y/no-static-element-interactions
         className={this.props.className}
         onBlur={this.onBlur}
         onClick={this.onClick}
         onFocus={this.onFocus}
+        onKeyDown={() => {}}
```

Comment:
Interesting. Does the jsx-a11y lint rule enforce having a keydown prop here? May be interesting to checking out if `onKeyDown={undefined}` works, as it could possibly be preferable.

Reply:
Yeah, or it might have been a warning. I was just shooting for a clean ES lint output. Definitely worth a follow up.